features/roommembermoderation/impl/src/main/kotlin/io/element/android/features/roommembermoderation/impl/RoomMemberModerationPresenter.kt

```diff
@@ -97,6 +99,7 @@ class RoomMemberModerationPresenter @Inject constructor(
                             unbanUserAsyncAction.value = AsyncAction.ConfirmingNoParams
                         }
                     }
+                    moderationActions.value = persistentListOf()
```

Comment:
I am not sure to understand the effect of this. It is supposed to close the bottom sheet as soon as an action is selected, to avoid that a bottom sheet is displayed at the same time as a confirmation dialog, but this is already the current behavior. Is there some other rationale?